src/platform/telink/BUILD.gn

```diff
@@ -58,6 +58,10 @@ static_library("telink") {
   public_deps = [ "${chip_root}/src/platform:platform_base" ]
   deps = [ "${chip_root}/src/platform/logging:headers" ]
 
+  public_configs = [
```

Comment:
I think the linux need for this even on Linux was questionable.
Could you please explain why this is needed?